messaging/src/main/java/org/axonframework/messaging/OnNextMessageStream.java

```diff
@@ -57,4 +56,20 @@ public Optional<Entry<M>> next() {
         next.ifPresent(onNext);
         return next;
     }
+
```

Comment:
Nit: Missing class level JavaDoc. But will add it, as I've also worked on this PR.